.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           registry-url: https://registry.npmjs.org/
           cache: 'pnpm'
       - run: |
-          pnpm install
+          pnpm --filter ./packages/** --filter query install
```

Comment:
is `--filter query` responsible for installing the top level packages?

Reply:
Yes, it's the name you have in your root `package.json`